<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewProfileWindow</class>
 <widget class="QWidget" name="ViewProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#ViewProfileWindow{background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:0.503, y2:0.306727, stop:0.0696517 rgba(169, 169, 169, 255), stop:0.890547 rgba(255, 255, 255, 255));}
#stackedWidget{ background-color: rgba(1,1,1,0);}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="ViewProfileTitle">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>20</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>View Profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="VPFnameLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
      </font>
     </property>
     <property name="text">
      <string>Forename:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="VPEmailLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="VPPasswordLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="VPSnameLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
      </font>
     </property>
     <property name="text">
      <string>Surname:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="VPFnameInputBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VPSnameInputBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VPEmailInputBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VPPasswordInputBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="VPBackButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteAccButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>330</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>41</green>
           <blue>41</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>41</green>
           <blue>41</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>41</green>
           <blue>41</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delete Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ApplyButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Apply Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="VPRevealDataButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>330</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Show Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="ProfileImage">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>67</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="VPImageLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
      </font>
     </property>
     <property name="text">
      <string>Profile Image:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="VPImageCombo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
